components/mpas-albany-landice/bld/namelist_files/albany_input.yaml

```diff
@@ -1,11 +1,29 @@
 %YAML 1.1
 ---
 ANONYMOUS:
+# In order to use ML, change Tpetra to Epetra in the following line, 
+# and "Preconditioner Type: MueLu" to " Preconditioner Type: ML" several lines below 
   Build Type: Tpetra
 
+  Problem:
+    LandIce Field Norm:
+      sliding_velocity_basalside:
+        Regularization Type: Given Value
+        Regularization Value: 1.0e-4
+    LandIce BCs:
+      BC 0:
+        Cubature Degree: 4
+        Basal Friction Coefficient:
+          Type: Power Law
+          Power Exponent: 1.0 #left blank to cause an error if not filled
```

Comment:
this comment is confusing

Reply:
@mperego can you help us w/ some clarification for this comment? Thanks.

Reply:
I think the comment is for standalone Albany where we do not have defaults for `"Power Exponent`" and so we throw an error if not provided. 
That's not the case with MALI. I would simply remove the comment.

Reply:
I will remove it. Thanks.